### PR TITLE
Big fix for WebExtensions Panel auto-resize problem

### PR DESCRIPTION
**Pull request by kartik95**
Issue : https://github.com/mdn/webextensions-examples/issues/90

I fixed the bug along with some extra modifications. Changes made : 

1. The size of the panel will adjust according to the length of the text.
2. The overflow of the text would break in new line. (Actual bug fixed)
3. Made history links clickable.
4. Beautified the extension panel.

### CODE REVIEW

**Comment by kartik95** [comment]
@andymckay

**Comment by wbamberg** [comment]
@kartik95 , @andymckay , I'm on holiday today, but will be back tomorrow and will look at this then :-).

**Comment by kartik95** [comment]
Sure. Also, will this issue carry a 'hacktoberfest' label?

**Comment by andymckay** [comment]
Looks good to me, thanks @kartik95. I gave it a quick spin and it seems to work just great.

File: history-deleter/history.html [hunk 1]
```diff
@@ -7,9 +7,11 @@
   </head>
 
   <body>
-    <p>History for this domain (limited to 5 results):</p>
+    <p id="history-title">History for <span id="domain1"></span> (Last 5 results) :</p>
+    <div></div>
     <p id="history"></p>
-    <p><a href="#" id="clear">Clear history for this domain</a></p>
+    <div></div>
+    <p><a href="#" id="clear">Clear history for <span id="domain2"></span></a></p>
```
Comment (kartik95) [inline history-deleter/history.html]: We can do that. Should I commit again?

Comment (andymckay) [inline history-deleter/history.html]: Sure, if you do a commit on the same branch it will just update this pull request.

Comment (kartik95) [inline history-deleter/history.html]: Great. I'll do it. Also, is this issue counted in Hacktoberfest?

Comment (andymckay) [inline history-deleter/history.html]: Sorry, I've no idea what Hacktoberfest is.

Comment (kartik95) [inline history-deleter/history.html]: Committed the new code. And about hacktoberfest : https://hacktoberfest.digitalocean.com

**Comment by wbamberg** [comment]
This looks good, but should we use textContent instead of innerHTML?

"For that reason, it is recommended you not use innerHTML when inserting plain text; instead, use node.textContent."

https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#Security_considerations

**Comment by kartik95** [comment]
Thanks for the advice. Made the required changes. @wbamberg

File: history-deleter/history.css [hunk 2]
```diff
@@ -1,4 +1,32 @@
...
-    width: 350px;
+    margin: 0.3em;
+    width: auto;
+    min-width: 250px;
+    max-width: 500px;
+    background-color: #f8f8f8;
 }
+
+div {
+	margin-left: 0.5em;
```
Comment (andymckay) [inline history-deleter/history.css]: minor nit, but these CSS properties in indented by different amounts, something consistent would be cool

Comment (kartik95) [inline history-deleter/history.css]: Okay. I have a formatter embedded in my sublime. Javascript Beautify. It indents to 4 spaces whereas your code is indented to 2 spaces. Can I use that?

Comment (kartik95) [inline history-deleter/history.css]: I just checked my code and it is indented properly. Also, did everything again according to the repo format. Still no changes. :confused:

Comment (kartik95) [inline history-deleter/history.css]: Got the mistake, my bad.

**Comment by wbamberg** [comment]
Thanks @kartik95 !

**Pull request merged**